api/board/board.controller.js

```diff
@@ -66,15 +71,13 @@ export async function addBoard(req, res) {
 export async function updateBoard(req, res) {
   try {
     const { boardId } = req.params
-    const boardToInsert = req.body
+    const updates = req.body
 
-    if (!boardId) {
-      return res.status(400).send('Bad request')
-    }
+    const existingBoard = req.board
```

Comment:
req.board? I dont think is right.. we send the board updates in the body, but I dont think we send the board in the request.

Reply:
you test in the postman?

Reply:
> req.board? I dont think is right.. we send the board updates in the body, but I dont think we send the board in the request.


We get it from the middleware premission that we have before it gets to the update route.. then we can grab it from there and use it

Reply:
> you test in the postman?

It suppose to work. It worked with postman